dl_lib/engine/defaults.py

```diff
@@ -66,7 +66,7 @@ def default_argument_parser():
     # PyTorch still may leave orphan processes in multi-gpu training.
     # Therefore we use a deterministic way to obtain port,
     # so that users are aware of orphan processes by seeing the port occupied.
-    port = 2 ** 15 + 2 ** 14 + hash(os.getuid()) % 2 ** 14
+    port = 2 ** 15 + 2 ** 14 + hash("User_name") % 2 ** 14
```

Comment:
hash("User_name") is a fix value, please don't do that.

Reply:
It's a fixed value i know, but i think is impossible to train on 8-GPU windows machine, I will find a way to get uid on windows.